katana.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import collections
+
+from core.notify import Notify
+
+
+class Katana:
+    def __init__(self):
+
+        arg_names = ['title', 'message', 'app_name', 'app_icon']        
+
+        # x, y, z e w são os parametros informados 
+
+        # Cria um dicionario de chave/valor ex: {title=x, message=y, app_name=z, app_icon=w}
+        args = dict(zip(arg_names, sys.argv[1:]))
+
+        # Cria uma lista do dicionario anterior ex: data(title=x, message=y, app_name=z, app_icon=w)
+        arg_list = collections.namedtuple('data', arg_names) 
+
+        # Percorre cada parabetro e atribui um valor default caso esteja sem valor, ex: data(title='', message='', app_name='', app_icon='')
+        args =  arg_list(*(args.get(arg, "") for arg in arg_names))  
+
+        self.title = args.title if args.title else "Katana"
+        self.message = args.message if args.message else "Hello World from Katana"
+        self.app_name = args.app_name if args.app_name else "Katana"
+        self.app_icon = args.app_name if args.app_name else "./assets/katana-logo.svg"
```

Comment:
```suggestion
        self.title = args.title or "Katana"
        self.message = args.message or "Hello World from Katana"
        self.app_name = args.app_name or "Katana"
        self.app_icon = args.app_name or "./assets/katana-logo.svg"
```

Acho que funciona assim, não testei. Já tinha visto em outros lugares assim. 

Achei também um [post no stackoverfow](https://stackoverflow.com/questions/3929433/does-python-have-an-or-equals-function-like-in-ruby)

Reply:
rodô liso :rocket: